index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Page</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <header>
+        <div class="status">
+            <span id="time">9:41</span>
+            <span id="icons">
+                <img src="assets/images/Cellular.png" alt="" class="icon">
+                <img src="assets/images/Wifi.png" alt="" class="icon">
+                <img src="assets/images/Battery.png" alt="" class="icon">
+            </span>
+        </div>
+        <div class="banner">
+            <h1 class="header">John Doe</h1>
+    
+            <nav>
+                <img src="assets/images/menuIcon.png" alt="" class="icon menu-icon">
+                <div class="nav-links" style="display: none;">
+                    <a href="" id="portfolio">Portfolio</a>
+                    <a href="" id="about">About</a>
+                    <a href="" id="contact">Contact</a>
+                </div>
+            </nav>
+        </div>
+    </header>
+    <main>
+        <section class="hero">
+            <span class="header">Hey There.
+                I’m Alex</span>
+            <span class="sub-header">I’am a Software Developer</span>
+            <p class="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+            <div class="social-icons">
+                <a href="" class="social-link">
+                    <img src="assets/images/githubIcon.png" alt="" class="icon">
+                </a>
+                <a href="" class="social-link">
+                    <img src="assets/images/linkedin.png" alt="" class="icon">
+                </a>
+                <a href="" class="social-link">
+                    <img src="assets/images/twitterIcon.png" alt="" class="icon">
```

Comment:
- [ ] Kindly make sure all the social links are present.

![image](https://user-images.githubusercontent.com/92215878/204813706-3bcc4670-bc35-4687-acd9-dcf9f1e65e2c.png)

Reply:
fixed it